packages/quantic/cypress/e2e/default-2/search-box/search-box.cypress.ts

```diff
@@ -0,0 +1,634 @@
+import {configure} from '../../../page-objects/configurator';
+import {
+  InterceptAliases,
+  interceptSearch,
+  mockQuerySuggestions,
+} from '../../../page-objects/search';
+import {scope} from '../../../reporters/detailed-collector';
+import {SearchBoxActions as Actions} from './search-box-actions';
+import {SearchBoxExpectations as Expect} from './search-box-expectations';
+
+const engineId = 'quantic-search-box-engine';
+const recentQueriesLSKey = `LSKey[c]${engineId}_quantic-recent-queries`;
+
+interface StandaloneSearchBoxOptions {
+  engineId: string;
+  placeholder: string;
+  withoutSubmitButton: boolean;
+  numberOfSuggestions: number;
+  textarea: boolean;
+  disableRecentQueries: boolean;
+}
+
+const expectedAriaLiveMessage = (suggestionsCount: number) => {
+  return `${suggestionsCount} search suggestions found, to navigate use up and down arrows.`;
+};
+
+const variants = [
+  {variantName: 'default', textarea: false},
+  {variantName: 'expandable', textarea: true},
+];
+
+function setRecentQueriesInLocalStorage(recentQueries: String[]) {
+  window.localStorage.setItem(
+    recentQueriesLSKey,
+    JSON.stringify(recentQueries)
+  );
+}
+
+const defaultOptions: Partial<StandaloneSearchBoxOptions> = {
+  engineId,
+};
+
+describe('quantic-search-box', () => {
+  const pageUrl = 's/quantic-search-box';
+
+  function visitSearchBox(
+    options: Partial<StandaloneSearchBoxOptions> = defaultOptions
+  ) {
+    interceptSearch();
+    cy.visit(pageUrl);
+    configure(options);
+    cy.wait(InterceptAliases.Search);
+  }
+
+  variants.forEach(({variantName, textarea}) => {
```

Comment:
```suggestion
  const variants = [
    {variantName: 'default', textarea: false},
    {variantName: 'expandable', textarea: true},
  ];
  variants.forEach(({variantName, textarea}) => {
```
Do you mind putting the variants declaration right next to its usage? 
It makes it hard to understand having to go up to find what the different `variants` are.